_data/versions.yml

```diff
@@ -1,3 +1,257 @@
+v2.2:
+- title: v2.2.0
+  note: |
+    12-May 2017
+    
+    This release includes new versions of most of the key Calico components. Highlights include
+    * New version of Bird for BGP multipath support
+    * New version of GoBGP integration with IPIP support and other fixes
+    * A new CNI plugin, supporting the latest version of the CNI spec
+    * Further improvements to the Kubernetes datastore driver, bringing beta support for BGP routing.
+    * Some significant improvements to Felix performance and occupancy.
+
+    For more details see the [blog post](https://www.projectcalico.org/project-calico-2-2-released/) or the component release notes linked below.
+
+    There are two known issues with this release (that also affect previous releases). These issues will be addressed in v2.2.1 to be released shortly.
+    1. Some users may experience route flapping when using IPIP - For [more details](https://github.com/projectcalico/bird/pull/37)
```

Comment:
Should this read `See [issue](https://github.com/projectcalico/bird/pull/37) for more details`

Same below.